CRM/Contact/Form/Edit/TagsAndGroups.php

```diff
@@ -102,7 +102,7 @@ public static function buildQuickForm(
           }
 
           if ($groupElementType == 'select') {
-            $groupsOptions[$id] = $group['title'];
+            $groupsOptions[$id] = $group;
```

Comment:
I just have one more thing: line 109 just below also needs `title` removed, e.g. if you add Group to a profile.

Reply:
@demeritcowboy actually its fine, because its affecting only the group select widget now takes the array of information instead of just group title. The reason why the entire $group array is set, Whereas for advanced checkbox widget, it still needs only the group title as per old behaivour.

Reply:
Try adding group to a profile though and then using the profile.

Reply:
oops my bad ... updated now